src/couch_jobs/src/couch_jobs_fdb.erl

```diff
@@ -631,12 +631,12 @@ init_jtx({erlfdb_transaction, _} = Tx) ->
 
 
 ensure_current(#{jtx := true, tx := Tx} = JTx) ->
-    case get(?COUCH_JOBS_CURRENT) of
+    case get(?PDICT_CHECKED_MD_IS_CURRENT) of
         Tx ->
             JTx;
         _ ->
             JTx1 = update_current(JTx),
-            put(?COUCH_JOBS_CURRENT, Tx),
+            put(?PDICT_CHECKED_MD_IS_CURRENT, Tx),
```

Comment:
`PDICT_CHECKED_MD_IS_CURRENT` is `true|false`

Also I think maybe we shouldn't set `put(?PDICT_CHECKED_MD_IS_CURRENT` here and maybe `check(?COUCH_JOBS_CURRENT)` and `get(?PDICT_CHECKED_MD_IS_CURRENT)` and if either is set we skip it...

However,  I wonder until we have a common `couch_fdb` if it maybe simpler to just use a separate metadata key. Thinking some more here we could get in trouble overwriting the pdict keys here. Since we don't necessarily share the Db handle. We do want them to separately perform the metadata validation instead of bypassing.